test/integration/service_manual_guide_test.rb

```diff
@@ -66,4 +66,31 @@ class ServiceManualGuideTest < ActionDispatch::IntegrationTest
 
     assert page.has_link?('Give feedback about this page')
   end
+
+  test 'displays the most recent change history for a guide' do
+    setup_and_visit_example('service_manual_guide', 'with_change_history')
+
+    within('.change-history') do
+      assert page.has_content? 'This is our latest change'
+      assert page.has_content? 'This is the reason for our latest change'
+    end
+  end
+
+  test 'displays the change history for a guide' do
+    setup_and_visit_example('service_manual_guide', 'with_change_history')
+
+    within('.change-history__past') do
+      assert page.has_content? 'This is another change'
+      assert page.has_content? 'This is why we made this change and it has a second line of text'
+
+      assert page.has_content? 'Guidance created'
+    end
+  end
+
+  test 'omits the previous history if there is only one change' do
+    setup_and_visit_example('service_manual_guide', 'service_manual_guide')
+
+    refute page.has_content? 'Show all page updates'
+    refute page.has_css? '.change-history__past'
+  end
```

Comment:
We've covered off the error in the view, but I'd find it useful to have the test for the scenario in which the content_history is empty and we display neither the previous changes or the most recent change.

The absence of a major update will be increasingly odd over time so having the test is a kind "yeah, we know and this is the reason" will be useful.